### PR TITLE
Change output method

### LINKED ISSUE #34: Replace deprecated `set-output` commands
> Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

### LINKED ISSUE #63: Link the python script correctly
While using the latest version of the `bump-version` [action](https://github.com/bakdata/ci-templates/blob/v1.13.4/actions/bump-version/action.yaml) created by this [PR](https://github.com/bakdata/ci-templates/pull/57), following error is thrown: 

```
python: can't open file '/home/runner/work/working-dir/./actions/bump-version/scripts/get_version.py': 
[Errno 2] No such file or directory
```
Since we are giving a working directory [here](https://github.com/bakdata/ci-templates/blob/v1.13.4/actions/bump-version/action.yaml#L31-L35), the file `/actions/bump-version/scripts/get_version.py` is expected to be found (in the calling repository) in the given `working-directory`.

### PR DESCRIPTION
**Pull request by MichaelKora**
Fixes #63
Fixes #34
I removed the `scripts` dir.  Since the version [v1.13.2](https://github.com/bakdata/ci-templates/blob/v1.13.2/actions/bump-version/action.yaml) was working just fine, i am not seeing any reason why we should create an extra python file.

I solved [the initial issue](https://github.com/bakdata/ci-templates/issues/34) using [this approach](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/).

### CODE REVIEW

**Comment by disrupted** [comment]
if the Python script file is not used anymore, please delete it.

**Comment by disrupted** [comment]
Please redo the merge, it seems that you overwritten your changes.

File: actions/bump-version/action.yaml [hunk 1]
```diff
@@ -28,8 +28,8 @@ runs:
     - name: Bump version
       id: release-tag
       run: |
-        echo "oldTag=$(python ./actions/bump-version/scripts/get_version.py)" >> $GITHUB_OUTPUT
+        echo "oldTag=$(python -c "from configparser import ConfigParser; cfg = ConfigParser(); cfg.read('.bumpversion.cfg'); print(cfg['bumpversion']['current_version'])")" >> "$GITHUB_OUTPUT"
```
Comment (disrupted) [inline actions/bump-version/action.yaml:31]: did you confirm that this works? I am a bit concerned about the nested double quotes, that's why we extracted it to an external Python script

Comment (MichaelKora) [reply actions/bump-version/action.yaml:31]: yes it works: https://github.com/bakdata/github-playground/actions/runs/3788369900
https://github.com/bakdata/github-playground/blob/main/.github/workflows/helm-lint.yaml

**Review by disrupted — APPROVED** [review]

File: actions/bump-version/action.yaml [hunk 2]
```diff
@@ -28,8 +28,8 @@ runs:
...
         bump2version ${{ inputs.release-type }} --no-commit
-        echo "newTag=$(python ./actions/bump-version/scripts/get_version.py)" >> $GITHUB_OUTPUT
+        echo "newTag=$(python -c "from configparser import ConfigParser; cfg = ConfigParser(); cfg.read('.bumpversion.cfg'); print(cfg['bumpversion']['current_version'])")" >> "$GITHUB_OUTPUT"
```
Comment (raminqaf) [inline actions/bump-version/action.yaml:33]: According to this link https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
The `$GITHUB_OUTPUT` is not in quotes.

**Comment by raminqaf** [comment]
You mentioned that version [v1.13.2](https://github.com/bakdata/ci-templates/blob/v1.13.2/actions/bump-version/action.yaml) was working just fine. That is true. Because back then, we were using the `set-output`, which is now deprecated and replaced with the [environment files](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/). 
The reason I moved the python code to the script was that the new [environment files](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/) wrap the python code into double quotes and I was skeptical if this might break the CI template. Moreover, the code is hard to read and understand, and putting it in a file with a proper name (like `get-version`) describes its functionality.

File: actions/bump-version/action.yaml (hunk 2 above)
Comment (disrupted) [reply actions/bump-version/action.yaml:33]: probably actionlint would complain about potential word splitting if we remove the double quotes 😄
Probably it's fine either way

**Review by yannick-roeder — APPROVED** [review]

File: actions/bump-version/action.yaml (hunk 2 above)
Comment (yannick-roeder) [reply actions/bump-version/action.yaml:33]: For me it is also fine with the double quotes since we tested that it works

**Review by raminqaf — APPROVED** [review]

**Pull request merged**